types/react-router/index.d.ts

```diff
@@ -156,5 +156,5 @@ export function useLocation<S = H.LocationState>(): H.Location<S>;
 export function useParams<Params extends { [K in keyof Params]?: string } = {}>(): { [p in keyof Params]: string };
 
 export function useRouteMatch<Params extends { [K in keyof Params]?: string } = {}>(
-    path?: string | RouteProps,
+    path?: string | string[] | RouteProps,
```

Comment:
alternatively, we could change this to
```
path?: RouteProps | RouteProps["path"],
```
(and same for the `props` parameter of `matchPath`)

Reply:
Would you prefer to make that change? Would it make a difference?

Reply:
Nah, just something that crossed my mind as I was doing this. I’ll leave it as is if no one has any strong opinions.